courses/free-courses-pl.md

```diff
@@ -114,12 +114,15 @@
 
 ### Python
 
-* [Kurs online Python dla początkujących](https://www.flynerd.pl/tag/python-kurs) - Małgorzata Łyczywek AKA Rita (HTML)
+* [Jak nauczyć się programowania](https://www.youtube.com/watch?v=eB3r2NQwNi4&t=5300s)
```

Comment:
Don't invent titles! Use the title given by the resource.
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
remove time code from url